### ISSUE TITLE
TST context menu "Tree of Tabs" "Bookmark this tree..." creates empty folder 

### ISSUE THREAD

**Issue by infonut** [piroor/treestyletab#3375]
Just updated to Firefox 116.0.3 on  64-bit Windows 10 Home.
Tree tabs were there and just suddenly disappeared. Vanished. Empty sidebar. Apropos of nothing. 
That is not the report.

I had the tabs saved from "Bookmark this tree" so I restored them.
All good. Correct tree structure.
But now I have made changes and want to bookmark them again. TST acts accordingly and brings up the correct folder to save. I save it. It creates an empty folder.

The first time I did it, I could see the app cataloging all the tabs. But at the end the folder was empty.
The next time I did it, no cataloging occurred and ever since there is no cataloging and still only an empty folder is created.

![image](https://github.com/piroor/treestyletab/assets/19478187/36c9d852-9e5a-4add-8472-69b83d1fff2d)

![TST bookmark](https://github.com/piroor/treestyletab/assets/19478187/c6a5e453-49de-442a-991a-60b93d8c3732)

![image](https://github.com/piroor/treestyletab/assets/19478187/4b3ffca3-1c15-47e2-a6cc-a3b494f2ec2a)

FYI, I have this checked ("Allow to read and create bookmarks")

![image](https://github.com/piroor/treestyletab/assets/19478187/d2756eaa-1cd8-41f3-b81b-efebeae9330a)

I don't know if I inadvertently turned off/on a preference I should not have.
Or. If it is a bug from the Firefox Update.

Suggestions?

Most appreciated!

**Comment by infonut** [comment]
Hi,
This is still not working for me.
Anyone have any suggestions?
Thanks!

**Comment by infonut** [comment]
Still not working if anyone has any ideas....
Thank you!

**Comment by irvinm** [comment]
Do you have this enabled?

![image](https://github.com/piroor/treestyletab/assets/979729/834e8fff-834b-4533-9acf-d57b1bc75679)

I had no problems re-bookmarking a tree more than once, so I don't think it is a general problem.

![image](https://github.com/piroor/treestyletab/assets/979729/9431b174-cad6-49ef-9ba7-afb77e490541)

**Comment by piroor** [comment]
I couldn't reproduce this problem. Could you check the Browser Console (Ctrl-Shift-J) and the debugger console for TST (`about:debugging` => `This Firefox` => `Extensions` => `Tree Style Tab` => `Inspect` and choose `Console`) when you saw the empty folder problem? If you see any message in those consoles please post them here.

**Comment by infonut** [comment]
Hey. Sorry for the delay in response. I stopped checking after awhile when I didn't hear anything.
I finally figured it out. I have Firefox extension Default Bookmark Folder - this causes the tree style tabs create bookmarks to come up empty - actually, it creates a bizarre behavior where it dumps all the tabs in a Bookmark Toolbar folder.
Anyway, I didn't realize this Default Bookmark Folder extension was still active. When I turned it off,  everything works as it is supposed to.
Thanks for all the answers above!
